projects/plugins/jetpack/modules/shortcodes/css/quiz.css

```
@@ -54,3 +54,35 @@ div.jetpack-quiz pre {
 	padding: 0;
 }
 
+.jetpack-quiz-options {
+	margin-top: 12px;
+	text-align: right;
+}
+
+.jetpack-quiz-count {
+	color: #646970;
+	float: left;
+	font-size: 12px;
+}
+
+.jetpack-quiz-option-button {
+	cursor: pointer;
+	margin-left: 12px;
+}
+
+.jetpack-quiz-option-button:hover svg {
+	background: #3c434a;
+}
+
+.jetpack-quiz-option-button svg {
+	background: #000;
+	border-radius: 50%;
```

I would suggest adding a display to ensure things will be displayed nicely in themes that already style SVGs:

```suggestion
	border-radius: 50%;
	display: inline-block;
```